### ISSUE TITLE
Connection status isn't being properly updated when the network adapter is reset

### ISSUE THREAD

**Issue by pcenov** [learningequality/kolibri#10777]
## Observed behavior
This as a follow-up issue noticed in https://github.com/learningequality/kolibri/pull/10748
If I turn off network adapter and then turn it on, the status remains 'Not connected' until I manually restart the server.

## Expected behavior
The status 'Not connected' should get updated and if I attempt to edit the schedule while in a disconnected state I should see an indication for that.

## Steps to reproduce the issue
1. Install the latest build asset from https://github.com/learningequality/kolibri/pull/10748
2. Create a scheduled sync for a device
3. Observe the behavior of the Status value once the connection is stopped and then restarted.

## Video
https://github.com/learningequality/kolibri/assets/79847249/8970811a-eab2-49f7-9eeb-9461f2475de0

## Usage Details
Kolibri 0.16alfa14
Windows 10, Ubuntu - Chrome

**Comment by bjester** [comment]
@pcenov Could you wait at least 60 seconds after turning wifi back on to see if the connection status updates?

**Comment by pcenov** [comment]
Hi @bjester, no the connection status doesn't update after waiting for any amount of time but to further clarify, this issue is observed on both Ubuntu and Windows if I turn off the network adapter (not just the wi-fi as I have incorrectly written above).
For example for Ubuntu I am using the following option as it's on a virtual machine:

![2023-06-05_13-31-22](https://github.com/learningequality/kolibri/assets/79847249/94741bcf-4ae9-49ed-9f73-eb291c56382c)

I did double check today and if I simply stop my personal wi-fi network on my laptop and then turn it on again then the connection status does change correctly.

**Comment by bjester** [comment]
Ah toggling the network adapter is a new scenario

**Comment by marcellamaki** [comment]
After chatting with @bjester - we decided that if this in a new manifestation of an existing bug within the zeroconf handling, and not a regression, we can downgrade this to P2. A little bit of investigation should be done before this issue is taken up, and we can proceed accordingly.
